BGAnimations/ScreenOptionsService overlay/Support.lua

```diff
@@ -25,6 +25,12 @@ a.BeginCommand=function(self)
 	-- in which case we'll want to call ThemePrefs.Save() now
 	ThemePrefs.Save()
 
+	-- In case we switched into SRPG5 and had Rainbow Mode enabled, disable it.
+	if ThemePrefs.Get("VisualStyle") == "SRPG5" and ThemePrefs.Get("RainbowMode") == true then
+		ThemePrefs.Set("RainbowMode", false)
+		ThemePrefs.Save()
```

Comment:
I think we should just do this before the Save() two lines above, so that we don't need two Save() calls.

Reply:
Slipped my mind. Done.